src/components/modals/ListModal.tsx

```diff
@@ -104,7 +104,7 @@ export function ListModal<T>({
         onViewableItemsChanged={onViewableItemsChanged}
       />
       <GradientFadeOut />
-      {closeArrow ? <ModalCloseArrow onPress={handleCancel} /> : null}
+      {closeArrow ? <ModalFooter fadeOut onPress={handleCancel} /> : null}
```

Comment:
For the `fadeOut` to work correctly, we need to add `contentContainerStyle={{ paddingBottom: theme.rem(ModalFooter.bottomRem) }}`. This will ensure we can scroll the bottom-most list item *above* the close arrow. Otherwise, the arrow will always cover the bottom-most item.

Reply:
I have added an exported `ModalFooter.bottomRem = 2.5` constant in `ModalParts` and copied the same
```tsx
const scrollPadding = React.useMemo<ViewStyle>(() => {
    return {
      paddingBottom: theme.rem(ModalFooter.bottomRem),
    }
  }, [theme])
```
`contentContainerStyle` from the GUI